src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -173,8 +182,8 @@ export default function SelectNextPage() {
       title="SelectNext"
       intro={
         <p>
-          <code>SelectNext</code> is a presentational component which behaves
-          like the native <code>{'<select>'}</code> element.
+          <code>SelectNext</code> is a composite component which behaves like
+          the native <code>{'<select>'}</code> element.
```

Comment:
This part could do with an expansion at some point to explain how it differs from a `<select>`, as well as linking to the accessibility patterns it implements, specifically the [Select-only Combobox](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/).

Reply:
There's an issue around improving this documentation. I'll add a reference to this comment to make sure it's not forgotten.